parametric/apps/nodejs/servicer.js

```diff
@@ -58,9 +58,13 @@ class Servicer {
     InjectHeaders = (call, callback) => {
         const { request } = call;
         const span = this.spans[request.span_id];
-        const http_headers = {};
+        const http_headersDict = {};
+        const http_headers = [];
 
-        tracer.inject(span, 'http_headers', http_headers);
+        tracer.inject(span, 'http_headers', http_headersDict);
+        for (const [key, value] of Object.entries(http_headersDict)) {
+            http_headers.push({key: key, value: value});
+        }
 
         const ctx = span._spanContext
         console.log('InjectHeaders', http_headers, ctx.toTraceparent(), ctx._tags)
```

Comment:
Forgot this console.log was there. Could delete this while you're in there if you want. Not necessary anymore. 😅

Reply:
Removed both console.log statements in f7f2b19244143ecb77c63ac1fa090bda70492387 😄